api/src/main/java/org/jenkinsci/plugins/workflow/graph/FlowNode.java

```diff
@@ -163,6 +177,19 @@ public BallColor getIconColor() {
     protected abstract String getTypeDisplayName();
 
     /**
+     * Gets the function name for this type of node.
```

Comment:
Pretty vague Javadoc. I would have some idea what you meant, but be wrong, unless I were actually reading the implementations. This is in general not a “function name” at all, it is just a block of human-readable text that may or may not include a `StepDescriptor.functionName`.